packages/core/docs/commercetools/getting-started.md

```diff
@@ -61,7 +61,54 @@ setup({
       'view_products:vsf-ct-dev',
       'view_published_products:vsf-ct-dev'
     ]
+  },
+  i18n: {
+    useNuxtI18nModule: true
+  }
+}],
+i18n: {
```

Comment:
this is not a part of vsf module config, IMHO here there should be a list of config options for CT module, and the  nuxt-i18n should be described in the separate docs about internationaliation